From 538ce76a6dff39905fd5c558d96180aef08000ff Mon Sep 17 00:00:00 2001
From: TheDBP <TheDBP@users.noreply.github.com>
Date: Tue, 15 Sep 2026 07:40:03 -0400
Subject: [PATCH] SetupWizard: preselect 3-button navigation

The navigation page defaults to gestures, which draws a pill and never shows the
nav-bar icons this image ships. Preselect 3-button; gestures stay one tap away.
---
 res/layout/setup_navigation.xml                               | 4 ++--
 src/org/lineageos/setupwizard/NavigationSettingsActivity.java | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/layout/setup_navigation.xml b/res/layout/setup_navigation.xml
index f89de6b..1e174db 100644
--- a/res/layout/setup_navigation.xml
+++ b/res/layout/setup_navigation.xml
@@ -37,7 +37,7 @@
                         android:id="@+id/navigation_illustration"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        app:lottie_rawRes="@raw/lottie_system_nav_fully_gestural"
+                        app:lottie_rawRes="@raw/lottie_system_nav_3_button"
                         app:lottie_autoPlay="true"
                         app:lottie_loop="true" />
 
@@ -49,12 +49,12 @@
                         <RadioButton
                             android:id="@+id/radio_gesture"
                             android:text="@string/gesture_navigation"
-                            android:checked="true"
                             style="@style/SudRadioButton" />
 
                         <RadioButton
                             android:id="@+id/radio_sw_keys"
                             android:text="@string/navbar_navigation"
+                            android:checked="true"
                             style="@style/SudRadioButton" />
 
                     </RadioGroup>
diff --git a/src/org/lineageos/setupwizard/NavigationSettingsActivity.java b/src/org/lineageos/setupwizard/NavigationSettingsActivity.java
index a4deb59..9161e40 100644
--- a/src/org/lineageos/setupwizard/NavigationSettingsActivity.java
+++ b/src/org/lineageos/setupwizard/NavigationSettingsActivity.java
@@ -31,7 +31,7 @@ public class NavigationSettingsActivity extends BaseSetupWizardActivity {
 
     private SetupWizardApp mSetupWizardApp;
 
-    private String mSelection = NAV_BAR_MODE_GESTURAL_OVERLAY;
+    private String mSelection = NAV_BAR_MODE_3BUTTON_OVERLAY;
 
     private CheckBox mHideGesturalHint;
 
